hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/wildfly-10-jmx-exporter.yml

```diff
@@ -19,11 +19,33 @@
 startDelaySeconds: 1
 lowercaseOutputName: true
 lowercaseOutputLabelNames: true
+whitelistObjectNames: [
+  "jboss.as:path=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subsystem=undertow,servlet=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow,servlet=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-queue=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-topic=*",
+  "jboss.as:subsystem=datasources,data-source=*,statistics=*",
+  "jboss.as:subsystem=datasources,xa-data-source=*,statistics=*",
+  "jboss.as:subsystem=transactions",
+  "jboss.as:subsystem=undertow,server=*,http-listener=*",
+  "jboss.as:subsystem=undertow,server=*,https-listener=*"
+]
 rules:
 
   # CONFIG
 
-  - pattern: "^jboss.as<path=(.+)><>path:(.+)"
```

Comment:
I believe we want to keep the ^ in all these patterns because it will be more efficient. Pattern matcher will fail fast if the head of the string doesn't match - it doesn't have to search for a match through the string.

Reply:
I think it's already built into the pattern here:

https://github.com/prometheus/jmx_exporter/blob/master/collector/src/main/java/io/prometheus/jmx/JmxCollector.java#L170

But I put them back to make you happy.

Reply:
oh yeah! so it is. you didn't have to put it back :)

Reply:
Nerd.  OK, I took them back out, now merge this before you change your mind :)